release_notes.md

```diff
@@ -1,4 +1,63 @@
-## Release 18.5.2018
+## Release 1.4.2019 v1.2
+
+### New Features
+ - Support for cloud functions was added for deployment and testing
+ - WAW now supports [pattern entities](/doc/WAW_entities_doc.md#patterns) and [contextual entities](/doc/WAW_entities_doc.md#contextual-entities) (csv to json direction only)
+ - We employ `logging` module for logging
+ - Using [workspace_addjson.py](/scripts/workspace_addjson.py) script you are now able to add custom json structures to specified location in the workspace json
+ - It is now possible to deploy workspace by its name (by setting `workspace_name_unique` to true)
+ - [Dialog testing script](/scripts/evaluate_tests.py) now outputs `junit.xml` file with test results
+ - [requirements.txt](/requirements.txt) and [requirements_dev.txt](/requirements_dev.txt) were added to simplify installation process
+ - Configuration files was splitted to three ones: [common.cfg](/example/en_app/common.cfg) (common project settings), [private.cfg](/example/en_app/private.cfg.template) (credentials - not to push) and [builg.cfg](/example/en_app/build.cfg) (setting specific for the build - usualy not to push)
```

Comment:
Change proposals:
- "Sample configuration file was split into three: ..."
- private.cfg.template) (filled-in template contains credentials, not to be pushed)
- build.cfg) (settings specific for concrete build environment, usually not to be pushed)

Reply:
Changed, thank you!!